src/integrationTest/java/uk/gov/hmcts/darts/retention/controller/RetentionControllerGetAllPolicesTest.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.darts.retention.controller;
+
+import org.junit.jupiter.api.Test;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import uk.gov.hmcts.darts.authorisation.component.UserIdentity;
+import uk.gov.hmcts.darts.testutils.IntegrationBase;
+import uk.gov.hmcts.darts.testutils.stubs.SuperAdminUserStub;
+
+import java.net.URI;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.hmcts.darts.testutils.TestUtils.getContentsFromFile;
+
+@AutoConfigureMockMvc
+class RetentionControllerGetAllPolicesTest extends IntegrationBase {
+    @Autowired
+    private transient MockMvc mockMvc;
```

Comment:
Does transient have a purpose here?

Reply:
Morning Tony. Yeah. It serves no purpose. I used it to be consistent with other Test cases I looked at that extend integrationBase, however looking again I can see that many do not and that is probably correct, as it is 
superfluous.